apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts

```diff
@@ -37,6 +41,27 @@ async function getFileTransfer(file_transfer_id: string, user_id: string): Promi
   return file_transfer;
 }
 
+async function doTransfer(res: Response, editor: Editor, fileTransferId: string) {
+  const serverJob = await editor.prepareServerJob();
+  try {
+    await editor.executeWithServerJob(serverJob);
+  } catch {
+    res.redirect(res.locals.urlPrefix + '/edit_error/' + serverJob.jobSequenceId);
+    return;
+  }
+
+  await sqldb.queryOneRowAsync(sql.soft_delete_file_transfer, {
+    id: fileTransferId,
+    user_id: res.locals.user.user_id,
+  });
```

Comment:
Nit: since the return value of this query isn't used, consider removing the return value and replacing the query function with `queryAsync`.

Reply:
done in eee15899d